src/components/Sources/Sources.tsx

```diff
@@ -212,6 +212,7 @@ export const Sources = () => {
         title="Source"
         headers={headers}
         rows={rows}
+        height={90}
       />
```

Comment:
This is not needed. The Sources list has the correct height after you added the height as a prop to the commmon `List` component and used it in only `Connections` page.